lib/core/src/lib/form/components/widgets/date/date.widget.ts

```diff
@@ -81,8 +80,17 @@ export class DateWidgetComponent extends WidgetComponent implements OnInit, OnDe
         this.validateField();
         this.onFieldChanged(this.field);
     }
-    private patchFormControl(): void {
+
+    updateReactiveFormControl(): void {
+        this.updateFormControlState();
+        this.validateField();
+    }
+
+    private setFormControlValue(): void {
         this.dateInputControl.setValue(this.field.value, { emitEvent: false });
+    }
+
+    private updateFormControlState(): void {
         this.dateInputControl.setValidators(this.isRequired() ? [Validators.required] : []);
```

Comment:
Can we use custom validators in date widget? 🤔

Reply:
as default option we use only required from default validators. for other cases, like min/max constraints, we handle it in the handleErrors method, and its needed to be compatible with existing form structure + other non-reactive form widgets.